lintro/tools/core/install_strategies/npm_strategy.py

```
@@ -0,0 +1,109 @@
+            return None
+        if env.has("brew") and tool_name in BREW_FORMULA_NAMES:
+            return None
+        return "bun/npm not available (install Node.js first)"
+
+    def install_hint(
+        self,
+        env: InstallEnvironment,
+        tool_name: str,
+        tool_version: str,
+        install_package: str | None,
+        install_component: str | None,
```

Ruff (0.15.7)

[warning] Unused method argument: `install_component`

(ARG002)